### PR TITLE
Changing model options, references, style and hyperlinks

### PR DESCRIPTION
**Pull request by vlasovv**
*Please include a short description of enhancement here*

**I hereby confirm that I have:**

- [x] Tested my code on my own machine
- [x] Followed the guidelines in the [Contributing Guide](https://opencobra.github.io/cobratoolbox/docs/contributing.html)
- [x] Selected `develop` as a target branch (top left drop-down menu)

*(Note: You may replace [] with [X] to check the box)*

### CODE REVIEW

File: tutorials/FBA/tutorial_FBA.m [hunk 1]
```diff
@@ -47,27 +47,45 @@
...
-clear modelRecon3model
+% check if Recon3 exists:
+% pathModel = '~/work/sbgCloud/data/models/unpublished/Recon3D_models/';
+% filename = '2017_04_28_Recon3d.mat';
+% load([pathModel, filename])
+% model = modelRecon3model;
+% clear modelRecon3model
+% and if not
+% select your own model, or use Recon2.0model instead filename='Recon3.0model';
+load('/fork-cobratoolbox/test/models/Recon2.0model.mat')
```
Comment (laurentheirendt) [inline tutorials/FBA/tutorial_FBA.m]: May you put here:
```
global CBTDIR
load([CBTDIR filesep 'test' filesep 'models' filesep 'Recon2.0model.mat']);
```

Comment (vlasovv) [inline tutorials/FBA/tutorial_FBA.m]: Ok I will change

Comment (vlasovv) [inline tutorials/FBA/tutorial_FBA.m]: Should I change to the new branch or on this?

**Comment by syarra** [comment]
Thanks @vlasovv!

**Pull request merged**